### PR TITLE
Fixes #31446 - Copy Content Views UI

### PR DESCRIPTION
**Pull request by sjha4**
The copy flow is functional with this PR.
Working on:
- [x] Tests

### CODE REVIEW

**Comment by johnpmitsch** [comment]
Works well and code looks good, will take another look after the tests are added!

File: webpack/scenes/ContentViews/Table/tableDataGenerator.js [hunk 1]
```diff
@@ -27,7 +27,7 @@ const buildRow = (contentView, openColumn) => {
     id, composite, name, environments, versions, last_published: lastPublished,
   } = contentView;
   const row = [
-    { title: <ContentViewIcon composite={composite ? true : undefined} /> },
+    { title: <ContentViewIcon composite={composite ? true : undefined} />, props: { id, name } },
```
Comment (johnpmitsch) [inline webpack/scenes/ContentViews/Table/tableDataGenerator.js:30]: @sjha4 is this still needed? Looks like the info is coming from the row data that is set up below.

Comment (sjha4) [reply webpack/scenes/ContentViews/Table/tableDataGenerator.js:30]: Right..This is not needed..

File: webpack/scenes/ContentViews/Copy/__tests__/copyContentView.test.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  fireEvent.change(getByLabelText('input_name'), { target: { value: 'cv copy' } });
+
+  getByLabelText('copy_content_view').click();
+  // Form closes it self on success
+  await patientlyWaitFor(() => {
+    expect(queryByText('Name')).not.toBeInTheDocument();
+  });
+
+  assertNockRequest(copyscope, done);
+});
```
Comment (johnpmitsch) [inline webpack/scenes/ContentViews/Copy/__tests__/copyContentView.test.js:53]: One thing I just noticed - this test is pretty similar to the other one, I'm fine with this test being the only test and checking multiple things during the test (the api call is made and the form is closed), these longer tests make sense in functional testing IMO

Comment (sjha4) [reply webpack/scenes/ContentViews/Copy/__tests__/copyContentView.test.js:39]: Updated..

**Review by johnpmitsch — APPROVED** [review]

**Pull request merged**